packages/core/src/components/HeaderIconLinkRow/IconLinkVertical.tsx

```diff
@@ -67,6 +72,19 @@ export function IconLinkVertical({
     );
   }
 
+  if (action) {
+    return (
+      <Link
+        className={classnames(classes.link, classes.linkStyle)}
+        href={href}
+        {...props}
+      >
+        {icon}
+        {action}
+      </Link>
+    );
+  }
+
   // Absolute links should not be using RouterLink
   if (href?.startsWith('//') || href?.includes('://')) {
```

Comment:
This should already be handled by https://github.com/backstage/backstage/blob/6a1183a349e0bb06adae4cdadaee9954ce679c77/packages/core/src/components/Link/Link.tsx#L38, so could remove this logic now that we move it to core